pandas/core/reshape/reshape.py

```diff
@@ -30,9 +30,8 @@
 import pandas.core.algorithms as algos
 from pandas._libs import algos as _algos, reshape as _reshape
 
-from pandas.core.frame import _shared_docs
-from pandas.util._decorators import Appender
 from pandas.core.index import Index, MultiIndex, _get_na_value
+from pandas.core.reshape.melt import melt
```

Comment:
this import should not be necessary, instead directly import from the new module